assets/eip-4361/Notes.md

```diff
@@ -0,0 +1,2 @@
+## Reference Implementation
+The reference implementation is in TypeScript and available at [https://github.com/spruceid/siwe](https://github.com/spruceid/siwe). It is also available on [NPM](https://www.npmjs.com/package/siwe).
```

Comment:
Sorry if I wasn't clear, but generally we don't allow external links (even in the assets directory.)

What I meant was that you can include the reference implementation directly (not as a submodule) to this directory.

Reply:
You can't actually link to a directory like this.  You have to link to a specific file (there is no directory listing capabilities in the rendered page).

Reply:
Hey @MicahZoltu in this case would it be best practice to link to the readme in the assets folder?

Reply:
That is one option.  My general recommendation is to trim the reference implementation down to one file if possible.  An ideal reference implementation focuses exclusively on the stuff that matters for a standard, and doesn't include a bunch of unrelated things that aren't critical for understanding/following the standard.  When this is done, one can *usually* get the reference implementation down to something pretty small.  If the reference implementation is still bigger than one file after this, it usually means the EIP is trying to do too much and is a good indication that perhaps it should be split up into multiple smaller EIPs that address more constrained issues.